### PR TITLE
Add environment variable and client validation telegram and readme step by step to run the project.

### PR DESCRIPTION
**Pull request by uppy19d0**
I made the project receive the variable in the form of enviroment to avoid that they are placed directly in the code and I made a validation to verify that the bot knows telegram, also I placed step by step information on how to run the project.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -9,23 +9,25 @@
 from pyrogram.types import InlineKeyboardMarkup, InlineKeyboardButton, Message
 from bs4 import BeautifulSoup
 import requests
+import os
+from telethon.sync import TelegramClient as Client
```
Comment (nuhmanpk) [inline main.py:13]: Does telethon Client and Pyrogram Client Do the same job?

File: README.md [hunk 2]
```diff
@@ -3,6 +3,43 @@
...
+
+bash
+Copy code
+python -m venv venv
+source venv/bin/activate  # On Unix/Linux systems
+Install Dependencies:
+Use pip to install the project's dependencies from the requirements.txt file.
+
+bash
+Copy code
```
Comment (nuhmanpk) [inline README.md:22]: Think you should Use md formatting here , ```Code ```

**Review by nuhmanpk — CHANGES_REQUESTED** [review]
Hey @uppy19d0  , Thanks for your time . I have mentioned some changes / Questions in your Pr , can we have a corrections on it?

File: main.py (hunk 1 above)
Comment (uppy19d0) [reply main.py:13]: Yes, but now I see that pyrogram.types documentation is better, I will remove telethon.sync.

File: README.md (hunk 2 above)
Comment (uppy19d0) [reply README.md:22]: Ok

**Review by nuhmanpk — APPROVED** [review]
Thanks

**Comment by uppy19d0** [comment]
@nuhmanpk  please add in the labels : 
hacktoberfest-accepted
hacktoberfest-approved  hacktoberfest
hacktoberfest2023

**Comment by nuhmanpk** [comment]
> @nuhmanpk please add in the labels : hacktoberfest-accepted hacktoberfest-approved hacktoberfest hacktoberfest2023

Think they are already added, you can track them in official 
hacktoberfest website.

**Pull request merged**